data/json/items/armor/jewelry.json

```diff
@@ -1178,375 +1178,32 @@
-        "description": "A necklace made of a fine gold chain with a Jehovah's Witnesses Bible symbol as a pendant.  It is a symbol of the faith of the Jehovah's Witnesses.  The pendant is shaped like a small black Bible, and reads: 'Jehovah's Witnesses, Proclaimers of God's Kingdom', and on the other side, 'No Blood Transfusion!  See medical directive in wallet or purse'.",
-        "color": "yellow",
-        "weight": 10
-      },
-      {
-        "id": "tau_necklace",
-        "name": { "str": "Tau Cross necklace" },
-        "description": "A necklace made of a fine gold chain with the Tau Cross symbol as a pendant.  It is a general symbol for Christian faith.  The Tau Cross is representative of salvation and of life, due in part to its association with the sparing of believers in the Bible.  As Tau is the last letter in the Hebrew alphabet, it symbolizes the revealed Word of God, its completion and perfection.  It also can be said to symbolize the Last Day…",
-        "color": "yellow",
-        "weight": 8
-      },
-      {
-        "id": "jewish_necklace1",
+        "id": "jewish_necklace",
         "name": { "str": "Star of David necklace" },
-        "description": "A necklace made of a fine gold chain with the Star of David symbol as a pendant.  It is a symbol of the Jewish faith.  The Star of David became officially associated with Judaism when it was chosen as the symbol to appear on a flag at the First Zionist Congress in 1897.  At that time, it became the symbol of the Jewish people, Zionism, and the Jewish communities throughout the world.",
-        "color": "yellow",
-        "weight": 12
-      },
-      {
-        "id": "jewish_necklace2",
-        "name": { "str": "Menorah necklace" },
-        "description": "A necklace made of a fine gold chain with the Menorah symbol as a pendant, where each branch is equivalent to each seven days of The Creation.  It is a symbol of the Jewish faith.  The symbol of the menorah dates to Jewish tradition: After the Maccabees' victory over the Greeks, there was only enough oil to burn a menorah (lamp) for one day in the Temple of Judea.  Miraculously, the oil burned for eight days, and lighting the Hanukkah menorah commemorates this miracle and Jewish holiday.",
-        "color": "yellow",
-        "weight": 9
-      },
-      {
-        "id": "odin_necklace1",
-        "name": { "str": "Horn of Odin necklace" },
-        "description": "A necklace made of a fine gold chain with the Horn of Odin symbol as a pendant.  It is a symbol of the Asatru faith.  The Horn of Odin is one of the most recognizable Norse symbols, consisting of three interlocking drinking horns and symbolizing poetic inspiration and wisdom.  It is also a nod to Viking drinking customs and the camaraderie derived from them.",
-        "color": "yellow",
-        "weight": 6
-      },
-      {
-        "id": "odin_necklace2",
-        "name": { "str": "Valknut necklace" },
```

Comment:
Get rid of this one.  Then get rid of the christian denominations though  I feel like theologically speaking some are different enough they should be kept but that gets into a lot.  Then keep everything after this except freemason and I suppose pastafarian.  If directions unclear on where to start I'm saying keep the odin necklace.

Reply:
Are these as practised and worshipped religions that common in NE, and common enough for people to own a necklace with that symbol? While Norse is a legit religion that has believers, I don't many people who wear Norse-related symbols are practising that faith.

Reply:
IMHO Christian denominations should stay